app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -73,4 +73,9 @@ object FeatureFlags {
      * Feature flag tracking: https://github.com/mozilla-mobile/fenix/issues/27759
      * */
     val storageMaintenanceFeature = Config.channel.isNightlyOrDebug
+
+    /**
+     * Enables the re-engagement notification feature.
+     */
+    val reEngagementNotificationFeature = Config.channel.isNightlyOrDebug
```

Comment:
We spoke earlier and decided that this should ride the trains / doesn't need to be limited to Nightly. Let's change before landing.

Reply:
Yes, I forgot.  I'll update this.  Thanks

Reply:
@rocketsroger on the other hand, if we want to land this before the "target URL" is finalized, then changing to `isDebug` is best. Otherwise, we can also wait. I perhaps optimistically assumed we get the URL today.

Reply:
Sure.  How about we keep it to isNightlyOrDebug so testing can start earlier?  I'll then update it once the URL is confirmed?

Reply:
Fine with me too, but then it will affect users. So, we should make sure the URL we use is OK.